package.json

```diff
@@ -17,7 +17,7 @@
     "bootstrap": "lerna bootstrap",
     "build": "./scripts/build.sh",
     "lint": "eslint packages/*/src/**/*.js packages/*/example/**/*.js packages/*/test/**/*.js",
-    "test": "cross-env BABEL_ENV=commonjs jest",
+    "test": "cross-env BABEL_ENV=commonjs jest --no-cache",
```

Comment:
eliminates weird jest behavior when it's cache prevents tests from failing on source code changes.